sotodlib/hwp/g3thwp.py

```diff
@@ -643,18 +643,20 @@ def _set_empty_axes(self, aman):
         aman.wrap_new('locked', shape=('samps', ), dtype=bool)
         aman.wrap_new('hwp_rate', shape=('samps', ), dtype=np.float16)
         aman.wrap_new('eval', shape=('samps', ), dtype=bool)
+        aman.wrap_new('hwp_err', shape=('samps', ), dtype=bool)
 
         return
 
-    def _write_empty_solution_h5(self, tod, output=None, h5_address=None):
-
+    def _write_empty_solution_h5(self, tod, output=None, h5_address=None, err=False):
+        
         logger.info('Writing empty solutions')
         # metadata loader requires a dets axis
         aman = sotodlib.core.AxisManager(tod.dets, tod.samps)
         self._set_empty_axes(aman)
         aman.timestamps[:] = tod.timestamps
+        if err: aman.hwp_err[:] = np.ones(len(tod.timestamps))
```

Comment:
I prefer something like
`aman.hwp_err[:] = np.full_like(tod.timestamps, err, dtype=bool)`

hwp_err is misleading as hwp angle uncertainty, so should we change to err_flag or err_msg?

But I think the error flag can be just a single bool value not an array.
Can we write a single bool value error flag to the h5 metadata file instead?
Or saving simple message like "No data", "data is too short", "Error in hwp angle calculation" would be more helpful?